standup_notes/__main__.py

```diff
@@ -94,6 +123,17 @@ def edit_note(day: date):
     else:
         # Note: It appears that the file will be saved regardless of what you do in your editor (in my case, vim).
         editor.edit(note, STANDUP_TEMPLATE.read())
+        # Note: Date will be inserted after the document has been finished being created
```

Comment:
I found a way of doing this without needing to read then write to the file that might be bit cleaner: https://stackoverflow.com/questions/5914627/prepend-line-to-beginning-of-a-file/5917395

Reply:
Way of editing is now much simplier